k8s/cassandra/apptest/deployer/schema.yaml

```diff
@@ -0,0 +1,3 @@
+properties:
+  testerImage:
+    type: string
```

Comment:
Add x-google-marketplace as well:

```yaml
testerImage:
  type: string
  x-google-property:
    type: IMAGE
```

Reply:
I understand that it's only about `x-google-property`, not `x-google-marketplace` related?